src/views/RandomView.vue

```diff
@@ -17,20 +17,63 @@ import { ref } from "vue"
       return booster.value = newBooster;
       })
   }
+
+  const getNewDrink = () => {
+    fetch("https://www.thecocktaildb.com/api/json/v1/1/random.php")
+    .then(response => {
+      console.log(response)
+      return response.json()
+      })
+    .then(data => {
+      const ingredientsList = [];
+      for (let i = 1; i < 16; i++) {
+        if (data.drinks[0][`strIngredient${i}`]) {
+          const ingredient = data.drinks[0][`strMeasure${i}`] + " " + data.drinks[0][`strIngredient${i}`]
+          ingredientsList.push(ingredient)
+        }
+      }
+      const newBooster = {
+        title: data.drinks[0].strDrink,
+        preparation: data.drinks[0].strInstructions,
+        ingredients: ingredientsList,
+        id:Date.now(),
+        type:"drink",
+        isFavorited:false
+      }
+      return booster.value = newBooster;
+  })
+      // const newBooster = {
+      //   text: data.text,
+      //   id: Date.now(),
+      //   type: "fact",
+      //   isFavorited: false,
+      // }
+      // return booster.value = newBooster;
+      // })
+  }
 </script>
 
 <template>
   <div class="random">
     <div class="button-container">
       <button @click="getNewFact">Give me a new fact</button>
       <button>Give me a new song</button>
-      <button>Give me a new drink</button>
+      <button @click="getNewDrink">Give me a new drink</button>
     </div>
-    <BoosterCard v-if= "booster" 
+    <BoosterCard v-if= "booster && booster.type ==='fact'" 
```

Comment:
Let's change the name of this component eventually but it's not extremely urgent!